src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -72,10 +72,33 @@ public void setPerson(ReadOnlyPerson target, ReadOnlyPerson editedPerson)
     }
     //@@author TanYikai
     /**
-     * Sorts the persons object in the list alphanumerically by name.
+     * Sorts the persons object according to the sortOption.
+     * 0,1,2,3,4 represents name, phone, email, address, remark respectively
      */
-    public void sort() {
+    public void sort(int sortOption) {
```

Comment:
A cleaner way of solving this without jamming the sort option into every `Person` is to have this take a `Comparator<Person>`, and use [Comparator.comparing](https://docs.oracle.com/javase/8/docs/api/java/util/Comparator.html#comparing-java.util.function.Function-) to create the Comparator for the required sort key in the command.